### PR TITLE
Package resource-pooling.1.1

### PR DESCRIPTION
**Pull request by jrochel**
### `resource-pooling.1.1`
Library for pooling resources like connections, threads, or similar
This package is derived from the module Lwt_pool from the lwt package, which implements resource pooling. With Resource_pool this package provides a modified version with additional features. Also there is a module called Server_pool that manages resource clusters, specifically a cluster of servers each with its own connection pool.

---
* Homepage: https://github.com/ocsigen/resource-pooling
* Source repo: git+https://github.com/ocsigen/resource-pooling.git
* Bug tracker: https://github.com/ocsigen/resource-pooling/issues

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

File: packages/resource-pooling/resource-pooling.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+homepage: "https://github.com/ocsigen/resource-pooling"
+dev-repo: "git+https://github.com/ocsigen/resource-pooling.git"
+bug-reports: "https://github.com/ocsigen/resource-pooling/issues"
+build: [["dune" "build" "-j" jobs]]
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune"
+  "lwt" {>= "2.4.7"}
+  "lwt_log"
+  "ocamlbuild" {build}
```
Comment (kit-ty-kate) [inline packages/resource-pooling/resource-pooling.1.1/opam]:
```suggestion
```
ocamlbuild doesn't seem to be needed anymore

File: packages/resource-pooling/resource-pooling.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+  "ocamlfind" {build}
```
Comment (kit-ty-kate) [inline packages/resource-pooling/resource-pooling.1.1/opam]:
```suggestion
```
ocamlfind doesn't seem to be needed anymore

File: packages/resource-pooling/resource-pooling.1.1/opam [hunk 3]
```diff
@@ -0,0 +1,25 @@
+opam-version: "2.0"
+maintainer: "Jan Rochel <jan@besport.com>"
+authors: [ "Jan Rochel (BeSport)" ]
+synopsis: "Library for pooling resources like connections, threads, or similar"
+description: "This package is derived from the module Lwt_pool from the lwt package, which implements resource pooling. With Resource_pool this package provides a modified version with additional features. Also there is a module called Server_pool that manages resource clusters, specifically a cluster of servers each with its own connection pool."
+license: "MIT"
+homepage: "https://github.com/ocsigen/resource-pooling"
+dev-repo: "git+https://github.com/ocsigen/resource-pooling.git"
+bug-reports: "https://github.com/ocsigen/resource-pooling/issues"
+build: [["dune" "build" "-j" jobs]]
```
Comment (kit-ty-kate) [inline packages/resource-pooling/resource-pooling.1.1/opam]:
```suggestion
build: [["dune" "build" "-p" name "-j" jobs]]
```
Always use `-p` when calling dune in releases

Comment (kit-ty-kate) [reply packages/resource-pooling/resource-pooling.1.1/opam]: Reason being that without it `--profile` is set to `dev` which enables some warnings as errors

File: packages/resource-pooling/resource-pooling.1.1/opam [hunk 4]
```diff
@@ -0,0 +1,25 @@
...
+depends: [
+  "ocaml" {>= "4.06"}
+  "dune"
```
Comment (kit-ty-kate) [inline packages/resource-pooling/resource-pooling.1.1/opam]: As a side note, for the next release you should think about putting the `dune-project` file into the repository as it is required for dune 2.0 and also helps check (for us) which minimal version of dune is needed.

Comment (jrochel) [reply packages/resource-pooling/resource-pooling.1.1/opam]: Many thanks! All suggestions have been applied.

Comment (jrochel) [reply packages/resource-pooling/resource-pooling.1.1/opam]: Done. Sorry for all the trouble.

Comment (kit-ty-kate) [reply packages/resource-pooling/resource-pooling.1.1/opam]: No worries

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**